tools/clang/test/HLSLFileCheck/hlsl/workgraph/noinput_rdat.hlsl

```diff
@@ -0,0 +1,371 @@
+// CHECK:      ShaderKind: Node
+// CHECK:      PayloadSizeInBytes: 0
+// CHECK:      AttributeSizeInBytes: 0
+// CHECK:      FeatureInfo1: 0
+// CHECK:      FeatureInfo2: 0
+// CHECK:      ShaderStageFlag: (Node)
+// CHECK:      MinShaderTarget: 0xf0068
+// CHECK:      MinimumExpectedWaveLaneCount: 0
+// CHECK:      MaximumExpectedWaveLaneCount: 0
+// CHECK:      ShaderFlags: 0 (None)
+// CHECK:      Node: <2:NodeShaderInfo> = {
+// CHECK:        LaunchType: Thread
+// CHECK:        GroupSharedBytesUsed: 0
+// CHECK:        Attribs: <{{[0-9]+}}:RecordArrayRef<NodeShaderFuncAttrib>[2]>  = {
+// CHECK:          [0]: <4:NodeShaderFuncAttrib> = {
+// CHECK:            AttribKind: ID
+// CHECK:            ID: <2:NodeID> = {
+// CHECK:              Name: "noinput_thread"
+// CHECK:              Index: 0
+// CHECK:            }
+// CHECK:          }
+// CHECK:          [1]: <1:NodeShaderFuncAttrib> = {
+// CHECK:            AttribKind: NumThreads
+// CHECK:            NumThreads: <0:array[3]> = { 1, 1, 1 }
+// CHECK:          }
+// CHECK:        }
+// CHECK:        Outputs: <RecordArrayRef<IONode>[0]> = {}
+// CHECK:        Inputs: <RecordArrayRef<IONode>[0]> = {}
+// CHECK:      }
+// CHECK:    }
+// CHECK:    <3:RuntimeDataFunctionInfo{{.*}}> = {
+// CHECK:      Name: "noinput_mesh"
+// CHECK:      UnmangledName: "noinput_mesh"
+// CHECK:      Resources: <RecordArrayRef<RuntimeDataResourceInfo>[0]> = {}
+// CHECK:      FunctionDependencies: <string[0]> = {}
+// CHECK:      ShaderKind: Node
+// CHECK:      PayloadSizeInBytes: 0
+// CHECK:      AttributeSizeInBytes: 0
+// CHECK:      FeatureInfo1: 0
+// CHECK:      FeatureInfo2: 0
+// CHECK:      ShaderStageFlag: (Node)
+// CHECK:      MinShaderTarget: 0xf0068
+// CHECK:      MinimumExpectedWaveLaneCount: 0
+// CHECK:      MaximumExpectedWaveLaneCount: 0
+// CHECK:      ShaderFlags: 0 (None)
+// CHECK:      Node: <3:NodeShaderInfo> = {
+// CHECK:        LaunchType: Mesh
+// CHECK:        GroupSharedBytesUsed: 0
+// CHECK:        Attribs: <{{[0-9]+}}:RecordArrayRef<NodeShaderFuncAttrib>[3]>  = {
+// CHECK:          [0]: <5:NodeShaderFuncAttrib> = {
+// CHECK:            AttribKind: ID
+// CHECK:            ID: <3:NodeID> = {
+// CHECK:              Name: "noinput_mesh"
+// CHECK:              Index: 0
+// CHECK:            }
+// CHECK:          }
+// CHECK:          [1]: <1:NodeShaderFuncAttrib> = {
+// CHECK:            AttribKind: NumThreads
+// CHECK:            NumThreads: <0:array[3]> = { 1, 1, 1 }
+// CHECK:          }
+// CHECK:          [2]: <2:NodeShaderFuncAttrib> = {
+// CHECK:            AttribKind: DispatchGrid
+// CHECK:            DispatchGrid: <4:array[3]> = { 4, 1, 1 }
+// CHECK:          }
+// CHECK:        }
+// CHECK:        Outputs: <RecordArrayRef<IONode>[0]> = {}
+// CHECK:        Inputs: <RecordArrayRef<IONode>[0]> = {}
+// CHECK:      }
+// CHECK:    }
+// CHECK:  }
+// CHECK:  RecordTable (stride = {{[0-9]+}} bytes) NodeIDTable[4] = {
```

Comment:
Checking the contents of this table and the rest of the tables here is redundant if you already checked the nested record dumps in the `FunctionTable`.

Reply:
Removed